content/d-sgov/d-sgov-klasifikace/d-sgov-klasifikace-glosář.ttl

```diff
@@ -17,23 +17,293 @@ d-sgov-klasifikace:glosář a a-popis-dat-pojem:glosář, owl:Ontology, skos:Con
+  skos:altLabel "definice"@cs;
+  skos:definition "Delší text přesně definující klasifikaci a položky, které eviduje."@cs;
+  skos:inScheme d-sgov-klasifikace:glosář;
+  skos:prefLabel "definice klasifikace"@cs .
+
+d-sgov-klasifikace-pojem:akronym-klasifikace a skos:Concept;
+  skos:altLabel "akronym"@cs;
+  skos:definition "Zkratka klasifikace používaná i jako její identifikátor."@cs;
+  skos:inScheme d-sgov-klasifikace:glosář;
+  skos:prefLabel "akronym klasifikace"@cs .
+
+d-sgov-klasifikace-pojem:má-administrativní-platnost-klasifikace a skos:Concept;
+  skos:altLabel "má platnost"@cs;
+  skos:definition "Přiřazuje klasifikaci časový interval, ve kterém je platná. Platnost klasifikace určuje, v jakém časovém období byl, je nebo bude klasifikace platná."@cs;
+  skos:inScheme d-sgov-klasifikace:glosář;
+  skos:prefLabel "má administrativní platnost klasifikace"@cs .
+
+d-sgov-klasifikace-pojem:název-klasifikačního-schématu a skos:Concept;
+  skos:altLabel "název"@cs;
+  skos:definition "Název klasifikačního schématu je unikátním pojmenováním schématu, ze kterého vyplývá, jaké položky klasifikační schéma tvoří. Může být uveden ve více jazycích."@cs;
+  skos:inScheme d-sgov-klasifikace:glosář;
+  skos:prefLabel "název klasifikačního schématu"@cs .
+
+d-sgov-klasifikace-pojem:definice-klasifikačního-schématu a skos:Concept;
+  skos:altLabel "definice"@cs;
+  skos:definition "Delší text přesně definující klasifikační schéma a položky, které eviduje."@cs;
+  skos:inScheme d-sgov-klasifikace:glosář;
+  skos:prefLabel "definice klasifikačního schématu"@cs .
+
+d-sgov-klasifikace-pojem:akronym-klasifikačního-schématu a skos:Concept;
+  skos:altLabel "akronym"@cs;
+  skos:definition "Zkratka klasifikačního schématu používaná i jako jeho identifikátor."@cs;
+  skos:inScheme d-sgov-klasifikace:glosář;
+  skos:prefLabel "akronym klasifikačního schématu"@cs .
+
+d-sgov-klasifikace-pojem:má-administrativní-platnosti-klasifikačního-schématu a skos:Concept;
+  skos:altLabel "má platnost"@cs;
+  skos:definition "Přiřazuje klasifikačnímu schématu časový interval, ve kterém je schéma platné. Platnost schématu určuje, v jakém časovém období byl, je nebo bude schéma platné. Pro snazší strojové zpracování se doporučuje nespoléhat na to, že uživatel aplikuje platnost schématu i na jednotlivé položky. Je doporučeno platnost položek uvést přímo pro jednotlivé položky."@cs;
+  skos:inScheme d-sgov-klasifikace:glosář;
+  skos:prefLabel "má administrativní platnosti klasifikačního schématu"@cs .
+
+d-sgov-klasifikace-pojem:následuje-verzi-klasifikace a skos:Concept;
+  skos:altLabel "následuje verzi"@cs;
+  skos:definition "Propojuje konkrétní klasifikační schéma, které je verzí klasifikace, s jiným klasifikačním schématem, které je předchozí verzí klasifikace."@cs;
+  skos:inScheme d-sgov-klasifikace:glosář;
+  skos:prefLabel "následuje verzi klasifikace"@cs .
+
+d-sgov-klasifikace-pojem:nahrazuje-verzi-klasifikace a skos:Concept;
```

Comment:
jaká je sémantika tohoto vztahu? 

Znamená to, že "následuje verzi" nemusi zneplatnit tu předchozí, zatímco tento vztah tu předchozí verzi zneplatňuje?

Reply:
Ano, je to tak. Ale zapomněl jsem to dát do definice, takže jsem upravil definici obou.